Sources/Core/Private/Resolver.swift

```diff
@@ -269,6 +296,12 @@ class Resolver {
       return signature.call(objParameters)
     }
     
+    func setNSResolverIfNeeded(to obj: Any?) {
+      if let anObj = obj, component.injectToSubviews {
+        self.setNSResolver(to: anObj)
```

Comment:
`NSResolver(resolver: self).add(on: anObj)` - each object has its own

Reply:
Fixed. Create unique NSResolver instance for each object.